apache/zeppelin
# [ZEPPELIN-1246] In JDBCInterpreter.getScheduler, use getMaxConcurrentConnection instead of hardcoding maxConcurrency to 10

### What is this PR for?
In JDBCInterpreter.getScheduler, use getMaxConcurrentConnection instead of hardcoding maxConcurrency to 10


### What type of PR is it?
Improvement

### Todos

### What is the Jira issue?
[ZEPPELIN-1246] (https://issues.apache.org/jira/browse/ZEPPELIN-1246)

### How should this be tested?
Set zeppelin.jdbc.concurrent.max_connection to 2 and verify after starting two long-running queries, a third query is pending.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update?
No

* Is there breaking changes for older versions?
No

* Does this needs documentation?
No
